src/index.ts

```diff
@@ -39,6 +39,16 @@ function setupOpenMRS() {
         slot: 'nav-menu',
         load: getAsyncExtensionLifecycle(() => import('./nav-link'), options),
       },
+      {
```

Comment:
Why two?

Just register one extension component ID "patient-photo-widget" for an extension slot called "patient-photo".

You can use extension components multiple times and you can use extension slots multiple times. There is no restriction of having a single slot per microfrontend.